.travis.yml

```diff
@@ -10,14 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-before_install:
-  - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install icu4c; fi
```

Comment:
Should have been in all repos. That should have been the state wherever we had Travis builds on OSx. On the other hand, previously didn't have Travis builds on OSx in most repos.